lib/spark_api/models/contact.rb

```diff
@@ -45,6 +45,10 @@ def listing_carts(arguments = {})
         @listing_carts ||= ListingCart.collect(connection.get("/contacts/#{self.Id}/listingcarts", arguments))
       end
 
+      def messages(arguments = {})
+        @messages ||= Message.collect(connection.get("/contacts/#{self.Id}/messages", arguments))
```

Comment:
Another thing a little off with these methods is that if you call any of them multiple times with different arguments, you will still get the same results.